xgboost_ray/main.py

```diff
@@ -836,11 +893,14 @@ def train(params: Dict,
                 *args,
                 evals=evals,
                 ray_params=ray_params,
+                cpus_per_actor=cpus_per_actor,
+                gpus_per_actor=gpus_per_actor,
```

Comment:
maybe instead we can pass this in through a `actor_options` kwarg instead of 3 args?

Reply:
I changed it to just 1 additional arg for the placement group by setting the new cpus/gpus per actor back to ray_params.

Reply:
I'd actually prefer to leave it as is so it's clear as to what all of the arguments are. Plus this is an internal function so it should be fine to add these extra args. @krfricke any thoughts here?